docs-site/src/pages/docs/examples/plugins/example-usage.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import { controller } from '@brightinteractive/bright-js-framework'
+import { eventManager, EventManager } from '../plugins/EventManager'
+import { Button } from './widgets'
+
+@controller()
+export class ControlPanel extends React.PureComponent {
+  @eventManager
+  eventManager: EventManager
```

Comment:
So, to follow on from my question in the last PR.

In this example, if another component registered a handler onto the eventManager, any events fired by this component wouldn't trigger that handler?

Reply:
It would. Services != Dependencies. Services are unique, but dependencies provided by plugins are not. So you might use a service to handle form bindings, but a plugin to provide an event manager.

Reply:
I feel that it might be worth making this more explicit in the documentation.